recipe/meta.yaml

```diff
@@ -1,53 +1,50 @@
 {% set name = "cattrs" %}
-{% set version = "22.1.0" %}
-{% set sha256 = "94b67b64cf92c994f8784c40c082177dc916e0489a73a9a36b24eb18a9db40c6" %}
+{% set version = "23.1.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://github.com/python-attrs/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 9dbc7359c1ac2acdd011607218fd2aac26aab5fdaa51a7e162110d029e277ff3
 
 build:
   number: 0
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   skip: true  # [py<37]
 
 requirements:
   host:
     - python
     - pip
-    - poetry-core >=1.0.0
-    - wheel
+    - poetry-core >=1.1.0
   run:
     - python
-    - attrs >=20.1.0
-    - exceptiongroup  # [py<311]
-    - typing_extensions  # [py<38]
+    - attrs >=20
+    - exceptiongroup             # [py<311]
+    - typing_extensions >=4.1.0  # [py<311]
 
```

Comment:
any run_constrained ?
https://github.com/python-attrs/cattrs/blob/3cee857b51ed7c47bcfc72826ef70bd6de22a79f/pyproject.toml#L28-L34

Reply:
https://github.com/AnacondaRecipes/cattrs-feedstock/pull/5#discussion_r1387295023

Reply:
I assume the issues for the user would be apparently seen by package manager conflicts, but if we omit those altogether would not hide the underlaying incompatibilities in user env causing bugs which would be hard to notice, making things worse possibly ? that is just my 2 cents but we can discuss it

Reply:
It's better like that than introducing conflicts all over the place.

Reply:
I do not have a strong opinion, so I would go ahead with JC proposal, still can we can discuss some common approach in perseverance team meeting

Reply:
Ok, same here. Thank you!